libs/ui-lib/lib/common/components/operators/operatorSpecs.tsx

```diff
@@ -182,6 +182,24 @@ export const getOperatorSpecs = (
         category: categories[Category.VIRT],
         supportLevel: getFeatureSupportLevel('MTV'),
       },
+      {
+        operatorKey: OPERATOR_NAME_OSC,
+        title: 'OpenShift sandboxed containers',
+        featureId: 'OSC',
+        descriptionText: DESCRIPTION_OSC,
+        Requirements: () => (
+          <ExternalLink href={OSC_REQUIREMENTS_LINK}>
+            Learn more about the requirements for OpenShift sandboxed containers
+          </ExternalLink>
+        ),
+        Description: () => (
+          <>
+            {DESCRIPTION_OSC} <ExternalLink href={OSC_LINK}>Learn more</ExternalLink>
+          </>
+        ),
+        category: categories[Category.VIRT],
```

Comment:
can we remove the duplicate `category` prop from all items ?

Reply:
i'll take a look